railties/lib/rails/generators/database.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Rails
+  module Generators
+    module Database # :nodoc:
+      DATABASES = %w( mysql postgresql sqlite3 oracle frontbase ibm_db sqlserver )
+      JDBC_DATABASES = %w( jdbcmysql jdbcsqlite3 jdbcpostgresql jdbc )
+      DATABASES.concat(JDBC_DATABASES)
```

Comment:
Seems odd to mutate it right after defining it. Maybe swap definition order and use `+`?